node.cpp

```diff
@@ -0,0 +1,75 @@
+#include "node.h"
+#include "board.h"
```

Comment:
If you're using git on the command line you can run "git rm node.cpp" to get rid of this file.

 - [ ] Remove node.cpp